odd-platform-ui/src/components/DataEntityDetails/DatasetStructure/DatasetStructure.tsx

```diff
@@ -50,6 +57,15 @@ const DatasetStructureTable: React.FC<DatasetStructureTableProps> = ({
   isDatasetStructureFetching,
 }) => {
   const history = useHistory();
+  const [searchText, setSearchText] = React.useState<string>('');
+  const [scrollToIndex, setScrollToIndex] = useState(0);
```

Comment:
I think it's better to call `indexToScroll` than `scrollToIndex` because you are really storing the index in that variable.
Also, use common syntax - `React.useState();`